source/npm-util.js

```diff
@@ -61,3 +62,11 @@ exports.prereleaseTags = async packageName => {
 
 	return tags;
 };
+
+exports.verifyRecentNpmVersion = async () => {
+	const versions = JSON.parse(await execa.stdout('npm', ['version', '--json']));
```

Comment:
Just doing `npm --version` would be simpler. I also think we should have a separate method for getting the npm version.